### PR TITLE
New game page

### PR DESCRIPTION
**Pull request by pmwals09**
- Created new game form in React
- Created API controller tests
- Added form validations

### CODE REVIEW

File: app/controllers/api/v1/games_controller.rb [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+
   def index
     render json: Game.all
   end
+
+  def create
+    new_game = Game.new(game_params)
+    binding.pry
+    if new_game.save
+      binding.pry
```
Comment (jhayes00) [inline app/controllers/api/v1/games_controller.rb]: remove pries <3

File: app/javascript/react/components/NewGameForm.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          />
+
+        <label htmlFor="description">Game Description:</label>
+        <textarea
+          name="description"
+          id="description"
+          onChange={handleChange}
+          value={formVals.description}
+          />
+
```
Comment (safang99) [inline app/javascript/react/components/NewGameForm.js:69]: I would bring the "/>" on line 60 & 68 slightly to the left so they are in line with the opening "<".  For example the `ErrorList` on 50-52 is formatted correctly

File: app/javascript/react/components/NewGameForm.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        <label htmlFor="player_num">Number of Players:</label>
+        <input
+          type="text"
+          name="player_num"
+          id="player_num"
+          onChange={handleChange}/>
+
```
Comment (safang99) [inline app/javascript/react/components/NewGameForm.js:76]: I would also bring the "/>" at the end of 75 down to 76 to match

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -8,29 +8,27 @@
...
-        <%= link_to 'Sign Up', new_user_registration_path %> | 
-        <%= link_to 'Sign In', new_user_session_path %>
-      <%- end -%>
+  <div class="custom-header">
+    <div class="grid-x grid-padding-x align-middle">
+      <div class="cell small-3">
+        <%= link_to 'Add A Game', "/" %>
+      </div>
+
+
```
Comment (safang99) [inline app/views/layouts/application.html.erb:17]: lets remove one of the duplicate empty lines here

File: spec/controllers/api/v1/games_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+      after_count = Game.count
+
+      expect(after_count).to eq before_count
+    end
+
+    it "returns validation error json" do
+      post :create, params: bad_game_data, format: :json
+        api_response = JSON.parse(response.body)
+
+        expect(api_response["errors"]).to eq "Player number can't be blank"
```
Comment (safang99) [inline spec/controllers/api/v1/games_controller_spec.rb:64]: shift the line 62 & 64 slightly to the left to align with 61

**Pull request merged**